lib/services/auth_service.dart

```diff
@@ -60,7 +61,8 @@ class AuthService {
           user.serverAdress!, user.name!, user.password!);
     } catch (e) {
       if (user.serverAdress!.split(":").last != "8096" &&
-          user.serverAdress!.split(":").length == 2) {
+          user.serverAdress!.split(":").length == 2 &&
+          user.serverAdress!.split(":").first == 'http') {
```

Comment:
what if the server address is an ipv6 address with a port

Reply:
shouldn't make a difference right, http will always be the first element in the split